conf/modules/lidar_lite.xml

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="lidar_lite" dir="lidar">
```

Comment:
Hmm, I guess there will never be a good category for all sensors:-/ Personally I would be OK with leaving terranger in a separate folder, since it is rather unique. Technically it is not a lidar since it doesn't use laser (https://en.wikipedia.org/wiki/Lidar). What is your preference?